File: nafta.messenger.ios/nafta.messenger.ios/DataApi/Models/DTO/RegistrationModel.swift
//
//  RegistrationModel.swift
//  nafta.messenger.ios
//
//  Created by Yevhen Velizhenkov on 11/13/17.
//  Copyright © 2017 Yevhen Velizhenkov. All rights reserved.
//

import Foundation

struct RegistrationModel: Codable {
  var email = ""
  var password = ""
  var confirmPassword = ""
  var name = ""
  var surname = ""
  var sex = ""
  var bornDate: Date! = nil
  var phoneNumber = ""
  var registrationDate: Date! = nil
}

File: nafta.messenger.android/app/src/main/java/com/naftamessenger/data/models/dto/RegistrationModel.kt
package com.naftamessenger.data.models.dto

import com.naftamessenger.util.iso8601
import java.util.*

/**
 * Created by Bohdan on 19.11.2017
 */
class RegistrationModel {
	var email = ""
	var password = ""
	var confirmPassword = ""
	var name = ""
	var surname = ""
	var sex = "sex"
	var bornDate: String = Date().iso8601()
	var phoneNumber = "number"
	var registrationDate: String = Date().iso8601()
}